sw/device/exts/common/flash_link.ld

```diff
@@ -21,15 +21,12 @@ __DYNAMIC = 0;
  */
 INCLUDE hw/top_earlgrey/sw/autogen/top_earlgrey_memory.ld
 
-/* Reserving 32 bytes at the top of the RAM for test utils. */
-_test_reserved_size = 0x20;
-
 /**
  * The stack starts at the end of RAM right after the 32 reserved bytes and grows down.
  * It is zeroed out so a size must be provided.
  */
-_stack_size = 0x2000 - _test_reserved_size; /* ~8kb stack */
-_stack_start = ORIGIN(ram_main) + LENGTH(ram_main) - _test_reserved_size;
+_stack_size = 0x2000; /* 8kb stack */
+_stack_start = ORIGIN(ram_main) + LENGTH(ram_main);
```

Comment:
btw, there are two more linker scripts you missed:
- ./sw/device/mask_rom/mask_rom.ld
- ./sw/device/rom_exts/rom_ext.ld

Both have a now-unneeded test area.